app/src/main/java/com/seay/game_of_thrones/model/GoTCharacter.java

```diff
@@ -0,0 +1,44 @@
+package com.seay.game_of_thrones.model;
+
+public class GoTCharacter {
```

Comment:
So This is going to be a DTO. It is only used to transfer data from the network layer to the data layer. Let's rename it to `CharacterDTO`. `GoTCharacterDTO` is a fine name too, but once you instantiate an object, `characterDTO` is a better looking name than `goTCharacterDTO` but that's just my opinion.

Reply:
CharacterDTO is perfect. I just did GoTCharacter because Character is already a Java class